RunningOrder/Shared/Extensions/Cloudkit+Extensions.swift

```diff
@@ -132,6 +132,76 @@ extension CKFetchRecordsOperation {
     }
 }
 
+extension CKModifySubscriptionsOperation {
+    func publisher() -> AnyPublisher<([CKSubscription]?, [CKSubscription.ID]?), Error> {
+        let modifySubscriptions = PassthroughSubject<([CKSubscription]?, [CKSubscription.ID]?), Error>()
+        self.modifySubscriptionsCompletionBlock = { subscriptions, ids, error in
+            if let error = error {
+                modifySubscriptions.send(completion: .failure(error))
+                return
+            }
+
+            modifySubscriptions.send((subscriptions, ids))
+            modifySubscriptions.send(completion: .finished)
```

Comment:
not with Just because it is for a result we already have at the creation of the publisher. There is no `Single` type as in RxSwift, Just is the same between RxSwift and combine